data/hulls/ship_data.csv

```diff
@@ -7,50 +7,50 @@ Arbitrator,diableavionics_arbitrator,,Diable Avionics,diableavionics_booster,,30
 Coanda,diableavionics_coanda,Frontline Destroyer,Diable Avionics,diableavionics_damperwave,9,5500,700,4000,,350,100,,65,45,30,30,25,550,OMNI,,60,0.3,1,,,70,100,150,90,3,,9,45000,6,12,420,0.15,10,10,,diable_ship_bp,,0.25,2,3,,22
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 Gust,diableavionics_gust,Light Cruiser,Diable Avionics,diableavionics_assumeControl,14,7500,750,9500,,500,165,1,75,50,40,45,35,1500,OMNI,,60,0.35,1,,,145,230,120,120,3,,8,120000,4,16,480,0.1,20,20,,"diable_ship_bp,rare_bp",,0.5,2,3,,27
-Haze,diableavionics_haze,Cruiser,Diable Avionics,diableavionics_heavyflicker,15,10000,900,10750,,650,175,,60,45,40,35,25,1850,FRONT,,120,0.4,0.9,,,165,190,150,90,3,,8,130000,3,15,420,0.1,25,25,,"diable_ship_bp,rare_bp",,0.6,2,4,,28
-Storm,diableavionics_storm,Battlecarrier,Diable Avionics,microburn,18,9000,900,9000,,450,120,2,55,30,20,20,15,1700,OMNI,,90,0.35,0.9,,,255,425,275,200,4,,8,135000,3,15,480,0.1,26,26,"CARRIER, COMBAT","diable_ship_bp,rare_bp",,0.5,2,4,,32
+Haze,diableavionics_haze,Cruiser,Diable Avionics,diableavionics_heavyflicker,15,10000,900,10750,,650,175,,65,45,40,35,25,1850,FRONT,,120,0.4,0.9,,,165,190,150,90,3,,8,130000,3,15,420,0.1,25,25,,"diable_ship_bp,rare_bp",,0.6,2,4,,28
+Storm,diableavionics_storm,Battlecarrier,Diable Avionics,microburn,18,9000,900,9000,,450,120,2,55,30,20,20,15,1700,OMNI,,90,0.35,0.9,,,255,425,275,200,4,,8,135000,3,15,480,0.1,23,23,"CARRIER, COMBAT","diable_ship_bp,rare_bp",,0.5,2,4,,32
 Gulf,diableavionics_IBBgulf,Mod Cruiser,Diable Avionics,diableavionics_fluxredirection,22,10000,1000,16000,,800,200,2,60,40,30,30,25,2175,OMNI,,90,0.45,1,,,205,275,150,50,3.5,,8,200000,4,16,480,0.1,35,35,HIDE_IN_CODEX,restricted,,0.6,2,3,,30
 Daze,diableavionics_daze,Cruiser,Diable Avionics,diableavionics_fluxredirection,15,10000,1100,13000,,650,145,,50,25,20,17,15,1950,FRONT,,240,0.4,0.8,,,135,210,200,120,3,,8,125000,3,12,540,0.1,22,22,,"diable_ship_bp,rare_bp",,0.6,2,4,,28
 Rime (F),diableavionics_rime,Freighter,Diable Avionics,diableavionics_capacitors,10,6000,600,5500,,250,70,1,55,20,15,15,7,1750,OMNI,,150,0.5,1.1,,,70,150,600,500,3.5,,8,60000,5,20,360,0.1,10,10,CIVILIAN,diable_ship_bp,,0.5,2,3,,29
 Rime (P),diableavionics_rime_p,Q ship,Diable Avionics,microburn,10,6000,600,6000,,300,90,0,65,35,25,20,10,1750,OMNI,,150,0.5,1.1,,,70,150,400,150,3.5,,9,70000,5,20,420,0.1,14,14,,rare_bp,,0.5,2,3,,29
-Rime,diableavionics_rime_m,Recon Cruiser,Diable Avionics,diableavionics_capacitors,12,6000,600,6000,,300,110,2,65,35,25,20,10,1750,OMNI,,150,0.4,1,,,170,250,175,125,3.5,,9,90000,4,16,480,0.1,16,16,,diable_ship_bp,,0.5,2,3,,292
+Rime,diableavionics_rime_m,Recon Cruiser,Diable Avionics,diableavionics_capacitors,12,6000,600,6000,,300,110,2,75,35,25,20,10,1750,OMNI,,150,0.4,1,,,170,250,175,125,3.5,,9,90000,4,16,480,0.1,16,16,,diable_ship_bp,,0.5,2,3,,292
```

Comment:
Rime base speed increased to 75 (from 65). this buff needs to carry over to the Rime (P)